us-export-control.md

```diff
@@ -0,0 +1,111 @@
+# US export control process
+
+The United States imposes some controls over the export of cryptography.
+These controls include special allowances for open source software like Node.js.
+
+Best practice based on advice from the Node.js Foundation is to
+follow the "Technology and Software-Unrestricted" (TSU) licensing exception
+procedures. These procedures require the following:
+
+* Sending notification emails listing the locations
+  of our repositories that contain cryptographic software.
+* Validating that our licences are OSI-approved.
```

Comment:
Especially since this is a doc about US-specific stuff (and since it's our general style anyway), use the US spelling `licenses` rather than `licences`.